tsconfig.json

```diff
@@ -3,7 +3,8 @@
   "compilerOptions": {
     "outDir": "./lib",
     "resolveJsonModule": true,
-    "rootDir": "./src"
+    "rootDir": "./src",
+    "plugins": [{ "transform": "./messageTransformer/messageTransformer.ts" }]
```

Comment:
this property is only used by `ttypescript` / `ttsc`

https://github.com/cevek/ttypescript